pkg/server/server.go

```diff
@@ -363,7 +363,7 @@ func NewServerWithOpts(opts ...OpenFGAServiceV1Option) (*Server, error) {
 		requestDurationByDispatchCountHistogramBuckets: []uint{50, 200},
 		serviceName: openfgav1.OpenFGAService_ServiceDesc.ServiceName,
 
-		dispatchThrottlingCheckResolverEnabled:   serverconfig.DefaultCheckQueryCacheEnable,
+		dispatchThrottlingCheckResolverEnabled:   serverconfig.DefaultDispatchThrottlingEnabled,
```

Comment:
uff good catch.

Reply:
how did we miss this? should we update the test `dispatch_throttling_check_resolver_enabled`?

Reply:
Default values are constant and they have the same value (false).  I don't know if we can override constants in the UT.

Reply:
ah, i see. yeah not sure how to test for that..